### DISCUSSION TITLE
Smart Connections generates Connections based on dataview codes in each file

### DISCUSSION THREAD

**Discussion by zxedacus** [brianpetro/obsidian-smart-connections#228] [Q&A]
Hi Guys,

I have a specific dataview code that exists in every note in my obsidian.

eg.
```dataview
TABLE Topic, file.mday as Modified_Date, Project, Tags
WHERE contains(Topic, [[]])
```
At the same time, I have the same YAML fields in every note.

eg.
---
Tags:
Type: 
Category: 
Topic:
URL:
Note:
---

And due to the above, I believe Smart Connections think that my totally unrelated files are all related because they all have the same codes and YAML in them.

Are there anyway to prevent Smart Connections from taking YAML and Dataview codes into consideration when generating connections?

Thank you.

**Comment by brianpetro** [comment]
Hi @zxedacus

Thanks for using the Smart Connections and your feedback.

I understand this issue and have some thoughts on improving it.

For one, rendering the Dataview before embedding. And two, interpreting the yaml/meta variables in a way where their data is utilized but not necessarily in the same format, and excluding these when they're empty. 

I hope to ship both soon, but I can't say when exactly. Lots of cool updates coming soon. Some directly for this plugin and others as part of a new project I've been working on.

Brian 🌴
